### ISSUE TITLE
GolangCI failed with error message, Invalid output json

### ISSUE THREAD

**Issue by chandankumar4** [golangci/golangci-lint#817]
Thank you for creating the issue!

- [ ] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [ ] Yes, I've searched similar issues on GitHub and didn't find any.
- [ ] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

```bash
$ golangci-lint --version
golangci-lint v1.20.0
```

</details>

<details><summary>Config file</summary>

```bash
$ cat .golangci.yml
# No config file
```

</details>

<details><summary>Go environment</summary>

```bash
$ go version && go env
go version
    go version go1.13 linux/amd64
$ go env
    GO111MODULE=""
    GOARCH="amd64"
    GOBIN=""
    GOCACHE="/root/.cache/go-build"
    GOENV="/root/.config/go/env"
    GOEXE=""
    GOFLAGS=""
    GOHOSTARCH="amd64"
    GOHOSTOS="linux"
    GONOPROXY=""
    GONOSUMDB=""
    GOOS="linux"
    GOPATH="/go"
    GOPRIVATE=""
    GOPROXY="https://proxy.golang.org,direct"
    GOROOT="/usr/local/go"
    GOSUMDB="sum.golang.org"
    GOTMPDIR=""
    GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
    GCCGO="gccgo"
    AR="ar"
    CC="gcc"
    CXX="g++"
    CGO_ENABLED="1"
    GOMOD="/go/src/github.com/openebs/jiva/go.mod"
    CGO_CFLAGS="-g -O2"
    CGO_CPPFLAGS=""
    CGO_CXXFLAGS="-g -O2"
    CGO_FFLAGS="-g -O2"
    CGO_LDFLAGS="-g -O2"
    PKG_CONFIG="pkg-config"
    GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build006973412=/tmp/go-build -gno-record-gcc-switches"
```

</details>

<details><summary>Verbose output of running</summary>

```bash
$ golangci-lint run -v
# paste output here
```

</details>

**Anything else to know:**
![Screenshot from 2019-10-14 20-08-40](https://user-images.githubusercontent.com/28861964/66759801-8a1f1300-eebe-11e9-9a26-1703c744aecd.png)

Issue reference:  https://github.com/litmuschaos/chaos-exporter/pull/32

**Comment by vthiery** [comment]
Issue also raised [here](https://github.com/golangci/golangci-worker/issues/54)

**Comment by jirfag** [comment]
sorry, already fixed and released
